Jenkinsfile

```
@@ -298,6 +298,20 @@ pipeline {
         githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
       }
     }
+    stage('Snapshots') {
+      when {
+        allOf {
+          expression { return env.ONLY_DOCS == "false" }
+          anyOf {
+            branch 'master'
+            expression { return params.Run_As_Master_Branch }
+          }
+        }
+      }
+      steps {
+        doSnapshot()
```

Run in the `BASE_DIR` context

```suggestion
        dir("${BASE_DIR}") {
          doSnapshot()
        }
```